kwave/options/simulation_options.py

```diff
@@ -253,27 +253,22 @@ def option_factory(kgrid: "kWaveGrid", options: SimulationOptions):
                 else:
                     raise ValueError("Optional input ''pml_size'' must be a single numerical value.")
 
-        if kgrid.dim == 1:
-            options.pml_x_alpha = 2
+        if (kgrid.dim == 1):
             options.pml_x_size = options.pml_size if options.pml_size else 20
             options.plot_scale = [-1.1, 1.1]
-        elif kgrid.dim == 2:
-            options.pml_x_alpha = 2
-            options.pml_y_alpha = options.pml_x_alpha
-            if options.pml_size is None:
-                options.pml_x_size = 20
-                options.pml_y_size = 20
-            else:
-                options.pml_x_size = options.pml_size[0]
-                options.pml_y_size = options.pml_x_size
+        elif (kgrid.dim == 2):
+            if (options.pml_size is not None): 
+                if (len(options.pml_size) == kgrid.dim):
+                    options.pml_x_size, options.pml_y_size = np.asarray(options.pml_size, dtype=int).ravel()
+                else:
+                    options.pml_x_size, options.pml_y_size = (options.pml_size[0], options.pml_size[0])
+            else: 
+                options.pml_x_size, options.pml_y_size = (20, 20)
```

Comment:
I wonder if this bug exists in kWave-Matlab too. Have you had a chance to checkit?